pytest_html/plugin.py

```diff
@@ -260,7 +260,8 @@ def __init__(self, outcome, total=0, label=None,
 
             def generate_checkbox(self):
                 checkbox_kwargs = {'data-test-result':
-                                   self.test_result.lower()}
+                                   self.test_result.lower(),
+                                   'hidden': 'true'}
```

Comment:
This doesn't need to be in `checkbox_kwargs` as it can just be `hidden='true'` in the `self.checkbox` assignment.